CHANGELOG.txt

```diff
@@ -66,6 +66,7 @@
                 )
         # In 'creme_core.templatetags' :
             - In 'creme_widgets', the templatetag '{% widget_entity_hyperlink %}' has a new optional argument "target".
+        # Database indexes were added to improve general performance with large volumes of data.
```

Comment:
Please indicate the concerned models ("history"/"history lines" here), & we will extend the list with the next commits.